plugin/src/main/kotlin/com/github/triplet/gradle/play/internal/Plugins.kt

```diff
@@ -36,10 +36,11 @@ internal inline fun <reified T : Task> Project.newTask(
         block()
     }
 
+    val safeName = if (tasks.findByName(name) == null) name else "gpp" + name.capitalize()
```

Comment:
TODO(me): we shouldn't be using `findByName` because it forces the task to be created.

Reply:
On second thought, we're _expecting_ these tasks to be null. Using `named` means we have to throw and catch an exception every time we want to create a new task.